### PR TITLE
Do not attempt to install cloudsdk on master

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

File: jenkins/job-configs/jenkins-daily-maintenance.yaml [hunk 1]
```diff
@@ -8,7 +8,11 @@
     builders:
         - shell: |
             sudo apt-get update
-            sudo apt-get install -y google-cloud-sdk
+            if [[ "${HOSTNAME}" == "jenkins-master" || "${HOSTNAME}" == "pull-jenkins-master" ]]; then
+              echo Skipping google-cloud-sdk install...
```
Comment (fejta) [inline jenkins/job-configs/jenkins-daily-maintenance.yaml]: Done

**Comment by spxtr** [comment]
Super gross, but I'm okay with it. We could also try to be tricky with the Jenkins plugins, but that's even worse.

**Comment by fejta** [comment]
is that an LGTM?

**Comment by spxtr** [comment]
Yes

**Pull request merged**